packages/data-point/lib/entity-types/entity-hash/resolve.test.js

```diff
@@ -117,7 +117,7 @@ describe('entity.hash.pickKeys', () => {
       })
     })
   })
-  test('do nothing if mapKeys is empty', () => {
+  test('do nothing if mapKeys is empty | hash e2', () => {
```

Comment:
Can we add `It should` to the beginning of this description? I don't think we need the stuff from the `|` on.